### ISSUE TITLE
AWS IoT sdk - d

### ISSUE THREAD

**Issue by ahtashambaig** [espressif/esp-aws-iot#24]

**Comment by shahpiyushv** [comment]
@ahtashambaig , have you tried basic publish/subscribe example first to ensure that the MQTT communication itself (certificates, endpoint, client_id, policies, etc.) is fine?

**Comment by ahtashambaig** [comment]
Yes I did.. it is working also my program includes checks for testing my device connected to client or not ... Device gets connected to Endpoint and configured properly on both sides ... But on updating shadow it always return Update Timeout ... Also to be mentioned i have update my RX length
